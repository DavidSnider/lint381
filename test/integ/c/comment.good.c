struct foo;
// this is a comment with bad code: struct foo;
/* this is a block comment with bad code: struct foo; */
/*
 * this is a block comment with bad code: struct foo;
 */
