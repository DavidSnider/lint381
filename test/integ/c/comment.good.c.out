comment.good.c:1:8: error: Struct name 'foo' should be capitalized
